### DISCUSSION TITLE
Parent volunteer , not teacher- account category?

### DISCUSSION THREAD

**Discussion by smikes** [community/community#143593] [GitHub Education]
### Hi everyone,

How should I set up an account for a parent volunteer who is helping with a high school programming class, and is neither a teacher nor a student?

Context: teacher wants to accept help from a volunteer that could be a community member, parent, former student, etc. but does not fall into the category of “teacher”, “student”, or “school”.

**Comment by dhhoanggxz** [comment]
### Step-by-Step Instructions for Setting Up a Volunteer Account:

1. **Define Role and Permissions**
   - **Role**: Assign the volunteer as a "guest" or "volunteer" in the school's system (LMS or user management system).
   - **Permissions**: Grant only limited access, allowing them to view and engage with specific class materials and tools.

2. **Set Up the Account**
   - **Create an Account**: Use an organizational email if possible or allow the volunteer to register with a personal email for a guest account.
   - **Add to the System**: Log into the LMS (e.g., Google Classroom, Microsoft Teams) and add the volunteer as a “guest” or “viewer” role.

3. **Limit Access**
   - **Set Permissions**: Use user management settings to ensure that the volunteer only has "view" or specific access to class resources.
   - **Assign to Class**: Place the volunteer into a particular class or group where they will assist.

4. **Verification and Security**
   - **Collect Personal Information**: Obtain necessary details such as full name, ID, and contact information for verification.
   - **Enable MFA**: Require the volunteer to activate multi-factor authentication for account security.

5. **Provide Training and Share Policies**
   - **Training**: Offer brief guidance or documents on system usage and data protection.
   - **Policies**: Share school rules and data usage policies to ensure understanding and compliance.

6. **Maintain and Monitor Access**
   - **Set Expiry**: Establish an access expiration date to ensure the account is only active as long as needed.
   - **Monitor Activity**: Regularly review account usage to detect any irregularities.

7. **Ensure Compliance**
   - **Legal Checks**: Verify that the volunteer access complies with local privacy and education regulations.
   - **Agreement**: Have the volunteer sign an agreement acknowledging their compliance with the school's security and conduct policies.

These links are recommended and provides readers with easy access to more detailed, step-by-step instructions for specific systems.
[Google Classroom Help](https://support.google.com/edu/classroom/answer/6025224?hl=en) 
[Microsoft Teams Help](https://support.microsoft.com/en-us/teams)
